sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -28,7 +28,7 @@ import org.apache.spark.sql.catalyst.plans.logical.{Project, InsertIntoTable}
 import org.apache.spark.sql.execution.datasources.jdbc.JdbcUtils
 import org.apache.spark.sql.execution.datasources.{CreateTableUsingAsSelect, ResolvedDataSource}
 import org.apache.spark.sql.sources.HadoopFsRelation
-
+import org.apache.hadoop.io.compress.CompressionCodec
```

Comment:
Third party modules should be put above `org.apache.spark.*` modules.
see: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports